vm/systemSmartContracts/auction.go

```diff
@@ -183,13 +183,15 @@ func (s *stakingAuctionSC) changeRewardAddress(args *vmcommon.ContractCallInput)
 		return vmcommon.UserError
 	}
 
+	txData := "changeRewardAddress@" + hex.EncodeToString(registrationData.RewardAddress)
```

Comment:
for this, I have used something like this in genesis package
```GO
arguments := make([]string, 0, len(registrationData.BlsPubKeys) + 2)
arguments = append(arguments, changeRewardAddressFunction)
arguments = append(arguments, hex.EncodeToString(registrationData.RewardAddress))
for _, blsKey := range registrationData.BlsPubKeys {
    arguments = append(arguments, hex.EncodeToString(blsKey))
}
txData := strings.Join(arguments, "@")
```
As can be seen, "changeRewardAddress" might be a const for this.

Reply:
i would leave it like this. it is the same.

Reply:
I know, just it was clearer what you add to the arguments list.